src/runtime/components/nuxt-picture.ts

```diff
@@ -61,6 +62,15 @@ export default defineComponent({
       useHead({ link: [link] })
     }
 
+    if (process.server && process.env.prerender) {
+      const srcKey = nSources.value?.[1] ? 1 : 0
+      const sources = [
+        nSources.value[srcKey].src,
+        ...(nSources.value[srcKey].srcset || '').split(',').map(s => s.trim().split(' ')[0])
+      ].filter(s => s && s.includes('/_ipx/'))
+      appendHeader(useRequestEvent(), 'X-Nitro-Prerender', sources.join(','))
```

Comment:
Refactor: We shall create a utility (`prerenderImages(sources: string[]`) to reuse logic with img/picture components (https://github.com/nuxt/image/blob/200842abdf32aef9d34e39f845b0571103346f19/src/runtime/components/nuxt-img.ts#L89)